packages/qwik/src/core/shared/shared-serialization.ts

```diff
@@ -282,6 +283,7 @@ const inflate = (container: DeserializeContainer, target: any, typeId: TypeIds,
       computed.$untrackedValue$ = d[1];
       computed.$invalid$ = d[2];
       computed.$effects$ = d.slice(3);
+      (container as DomContainer).$scheduler$?.(ChoreType.QRL_RESOLVE, null, computed.$computeQrl$);
```

Comment:
also add `d[0].resolve()` here so it already tries to download it in this tick

+ add why it does this: to prevent anything from firing while computed qrls load so that they won't throw when not expected